src/components/panels/ToolheadControls/ZoffsetControl.vue

```diff
@@ -125,7 +79,7 @@
                                     small
                                     class="_btn-qs flex-grow-1 px-1"
                                     @click="sendBabyStepUp(offset)">
-                                    <v-icon v-if="index === 0 && !el.is.xsmall" left small class="mr-1 ml-1">
+                                    <v-icon v-if="index === 0 && !el.is.xsmall" left small class="mr-1 ml-n1">
```

Comment:
is this a typo?

Reply:
no, i tweaked the margins a bit. this looked better to me.